Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrieveAuditTransformer_g0.java

```diff
@@ -26,18 +26,19 @@
  */
 package gov.hhs.fha.nhinc.docretrieve.nhin;
 
-import ihe.iti.xds_b._2007.RetrieveDocumentSetRequestType;
-import ihe.iti.xds_b._2007.RetrieveDocumentSetResponseType;
 import gov.hhs.fha.nhinc.auditrepository.AuditRepositoryDocumentRetrieveLogger;
```

Comment:
Go back and see if there are unused imports here.  Pulled your code and saw a bunch.